container-toolkit/troubleshooting.md

```diff
@@ -119,4 +119,281 @@ Without this option, you might observe this error when running GPU containers:
 ``Failed to initialize NVML: Insufficient Permissions``.
 However, using this option disables SELinux separation in the container and the container is executed
 in an unconfined type.
-Review the SELinux policies on your system.
\ No newline at end of file
+Review the SELinux policies on your system.
+
+
+## Containers losing access to GPUs with error: "Failed to initialize NVML: Unknown Error"
+
+Under specific conditions, it’s possible that containerized GPU workloads may suddenly lose access to their GPUs. 
+This situation occurs when `systemd` is used to manage the cgroups of the container and it is triggered to reload any Unit files that have references to NVIDIA GPUs (e.g. with something as simple as a systemctl daemon-reload).
+
+When the container loses access to the GPU, you will see the following error message from the console output:
+
+```console
+Failed to initialize NVML: Unknown Error
+```
+
+The container needs to be deleted once the issue occurs.
+When it is restarted (manually or automatically depending if you are using a container orchestration platform), it will regain access to the GPU.
+
+The issue originates from the fact that recent versions of `runc` require that symlinks be present under `/dev/char` to any device nodes being injected into a container. Unfortunately, these symlinks are not present for NVIDIA devices, and the NVIDIA GPU driver does not provide a means for them to be created automatically.
+
+A fix will be present in the next patch release of all supported NVIDIA GPU drivers.
```

Comment:
Is there a specific version we can point to here? I assume this patch is already available